Api/CmsEntityConverterInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
Need to use strict_type declaration in there and in the next files.

declare(strict_types=1);